src/ogma/index.ts

```diff
@@ -309,10 +309,10 @@ export class LKOgma extends Ogma<LkNodeData, LkEdgeData> {
   /**
    * Return the list of filtered edges
    */
-  public getFilteredEdges(items?: Array<any>): EdgeList<LkEdgeData, LkNodeData> {
+  public getFilteredEdges(items?: Array<any>, filter: 'visible' | 'raw' | 'all' = 'raw'): EdgeList<LkEdgeData, LkNodeData> {
```

Comment:
it can be the same as getEdges (in that case we can simplify the code and have only one parameter "selector" ) but as I couldn't find in the code  any usage of getFilteredEdges with the usage of items I prefer to not do any refactoring now and change it later than just removing the any.

Reply:
ok